samples/client/petstore/typescript-node/default/api/storeApi.ts

```diff
@@ -86,7 +86,7 @@ export class StoreApi {
         (this.authentications as any)[StoreApiApiKeys[key]].apiKey = value;
     }
 
-    set accessToken(token: string) {
+    set accessTokenpetstore_auth(token: string) {
```

Comment:
this looks like it is generating a function name that isn't camelCase, and the bug looks like the collision was for one auth method that support both `basic` and `oauth`.

If we are going to include the name of the auth method we may as well include it on the basic too, or just suffix this one with `OAuth` to preserve function casing. Is there a scenario where these are colliding across different services?